cmd/clusterctl/cmd/rollout/pause.go

```diff
@@ -38,12 +38,14 @@ var (
 	pauseLong = templates.LongDesc(`
 		Mark the provided cluster-api resource as paused.
```

Comment:
Should we also update: https://cluster-api.sigs.k8s.io/clusterctl/commands/alpha-rollout.html?

Reply:
Good point, I was not aware of this file. Thanks

Reply:
I think that's fine

Reply:
Okay, perfect. Let me rebase